src/main/java/teammates/ui/template/InstructorCommentsCommentRow.java

```diff
@@ -0,0 +1,61 @@
+package teammates.ui.template;
+
+import teammates.common.datatransfer.CommentAttributes;
+
+public class InstructorCommentsCommentRow extends CommentRow{
+    private Boolean isInstructorAllowedToModifyCommentInSection;
+    private String typeOfPeopleCanViewComment;
+    private String editedAt;
+    private VisibilityCheckboxes visibilityCheckboxes;
+    
+    // Visibility Settings
+    private String showCommentsTo;
+    private String showGiverNameTo;
+    private String showRecipientNameTo;

```

Comment:
I hadn't checked on their usage, but are `visibilityCheckboxes` and the `show*To` variables not related?
It seems possible to combine them or something.

Also, is there a difference between `typeOfPeopleCanViewComment` and `showCommentsTo`?
Their names seem to refer to the same thing.